plugins/node/opentelemetry-instrumentation-aws-sdk/src/aws-sdk.ts

```diff
@@ -469,9 +478,18 @@ export class AwsInstrumentation extends InstrumentationBase<typeof AWS> {
               const promiseWithResponseLogic = resultPromise
                 .then(response => {
                   const requestId = response.output?.$metadata?.requestId;
+                  const httpStatusCode =
```

Comment:
Nit: Wondering if we should move this variable down 2 lines so that it is closer to the if block where it is used? That’s consistent with the `extendedRequestId` Block below, and then we can leave a blank space between these?

Reply:
Thanks for your comment. moved this variable